CHANGELOG.md

```diff
@@ -13,6 +13,7 @@ And please only add new entries to the top of this list, right below the `# Unre
 - **Breaking:** Bump `ndk` version to 0.6, ndk-sys to `v0.3`, `ndk-glue` to `0.6`.
 - Remove no longer needed `WINIT_LINK_COLORSYNC` environment variable.
 - **Breaking:** Rename the `Exit` variant of `ControlFlow` to `ExitWithCode`, which holds a value to control the exit code after running. Add an `Exit` constant which aliases to `ExitWithCode(0)` instead to avoid major breakage. This shouldn't affect most existing programs.
+- **Breaking:** On X11, device events are now ignored by default, use `EventLoop::set_filter_device_events` to opt-into device events again.
```

Comment:
I'm not sure making it disabled by default is a good idea while implementations for the other platforms don't exist.

Reply:
It's not really true, we have some device events on Wayland, like pointer motion information which is something used by e.g. games.

Reply:
The "correct" default would be to have device events be tied to window focus, which (AIUI) is what we get on Wayland. You'd then opt out of this behaviour on platforms where that's allowed.